qa/setup_packages.py

```diff
@@ -81,15 +85,37 @@ def clamp_index(self, idx, cuda_version=None):
             idx = 0
         return idx
 
-    def get_name(self, cuda_version=None):
+    def get_name(self, cuda_version=None, idx=None):
         """Retrives package name.
 
             Parameters
             ----------
             `cuda_version`: str, optional, default = None
                 Cuda version used for this query
+            `idx`: int
+                Index of name to retrive in case of specific version has different alias
         """
-        raise NotImplementedError
+        name = getattr(self.get_version(idx, cuda_version), "alias", None)
+        if name is None:
+            name = self.name
+        return name
+
+    def get_uninstall_names(self, cuda_version=None):
```

Comment:
So you specify the all possible uninstall strings, including the aliases, as in this place we don't care about the install version, right? Some of them will be just no-op?

Reply:
Yes. That is correct. For uninstalling we are not relying on the variant number so the script must return all possible variants. Maybe not the most beautiful but seems to be safest.

Reply:
Sure, seems fine, just wanted to check if I understood.